Clicker/Views/PollBuilder/MCPollBuilderView+Extension.swift

```diff
@@ -0,0 +1,170 @@
+//
+//  MCPollBuilderView+Extension.swift
+//  Clicker
+//
+//  Created by Kevin Chan on 12/6/18.
+//  Copyright © 2018 CornellAppDev. All rights reserved.
+//
+
+import IGListKit
+import Presentr
+
+extension MCPollBuilderView: ListAdapterDataSource {
+
+    func objects(for listAdapter: ListAdapter) -> [ListDiffable] {
+        var models: [ListDiffable] = [askQuestionModel]
+        models.append(contentsOf: mcOptionModels)
+        if let drafts = delegate?.drafts, !drafts.isEmpty {
+            draftsTitleModel = DraftsTitleModel(numDrafts: drafts.count)
+            if let draftsTitleModel = draftsTitleModel {
+                models.append(draftsTitleModel)
+            }
+            models.append(contentsOf: drafts)
+        }
+        return models
+    }
+
+    func listAdapter(_ listAdapter: ListAdapter, sectionControllerFor object: Any) -> ListSectionController {
```

Comment:
I wish objects for IGListKit did not have to be any :/

Reply:
:(